### ISSUE TITLE
Youtube rss feed doesn't work

### ISSUE THREAD

**Issue by Marcisbee** [MikaelPorttila/rss#43]
I tried using this package with youtube rss feed (https://www.youtube.com/feeds/videos.xml?channel_id=UCsXVk37bltHxD1rDPwtNM8Q) and it doesn't seem to be working.

I get this error:
```
TypeError: Cannot read properties of undefined (reading 'value')
    at mapAtomToFeed (https://deno.land/x/rss@0.5.5/src/mappers/mapper.ts:363:31)
    at toFeed (https://deno.land/x/rss@0.5.5/src/mappers/mapper.ts:29:14)
    at parseFeed (https://deno.land/x/rss@0.5.5/src/deserializer.ts:36:10)
```

**Comment by MikaelPorttila** [comment]
Hi @Marcisbee!
Thank you for report this. The issue should be fixed now with the 0.5.6 release, so please run the new bits and the problem should be gone. :)

BR
Micke

**Comment by Marcisbee** [comment]
@MikaelPorttila Very cool, everything works now!

Thanks!
